core/storage/base_model/gae_models_test.py

```diff
@@ -323,8 +323,8 @@ def setUp(self):
         self.model_instance = TestBaseHumanMaintainedModel(id=self.MODEL_ID)
         def mock_put(self):
             # type: (base_models.BaseHumanMaintainedModel) -> None
```

Comment:
Try to change to Python 3 typing wherever possible. I will typically comment for anything that's been touched during a PR (like this one), but at some point you might find it simpler to just do it once and for all for the codebase so that there is a consistent global standard.

Reply:
Hey Sean, We'll be changing the type annotations for files merged in PR1 in couple of weeks. I have changed all the files to be type annotated by this PR. Please wait till then, I had a talk with @sajalasati about this and planned this after my GSoC work. Thanks a lot!

Reply:
Hm OK -- for this case, I'm fine with that. But, just as a general note, if you make major changes to a file in a PR, that file should be typed properly as part of that PR.

Reply:
You should probably update the lint checks ASAP though to require the new Py3 annotations, so that other developers know to use them. Otherwise that just creates more work later on as the "not wanted" practices spread.

Reply:
I have also created an issue for this for tracking. See [here](https://github.com/oppia/oppia/issues/13602). Thanks!!